pkg/label/label.go

```diff
@@ -9,6 +9,17 @@ const (
 	// Cluster label is a new style label for ClusterID
 	Cluster = "giantswarm.io/cluster"
 
+	// ConfigMapType is a type of configmap used for tenant clusters.
+	ConfigMapType = "configmap.giantswarm.io/type"
```

Comment:
This new label is to identify the type of configmap. As different handling is needed.

Reply:
Doesn't look like conforming with fmt. If this is managed by `cluster-operator` it should be something like `cluster-operator.giantswarm.io/configmap-type`. I don't think labels should have any semantic meaning before `/`. This is just a scope.

Reply:
Good point. Done.